cli/hzn.go

```diff
@@ -368,15 +368,18 @@ Environment Variables:
 	deploycheckLong := deploycheckCmd.Flag("long", msgPrinter.Sprintf("Show policies and userinput used for the compatibility checking.")).Short('l').Bool()
 	policyCompCmd := deploycheckCmd.Command("policy", msgPrinter.Sprintf("Check policy compatibility."))
 	policyCompNodeArch := policyCompCmd.Flag("arch", msgPrinter.Sprintf("The architecture of the node. It is required when -n is not specified. If omitted, the service of all the architectures referenced in the deployment policy will be checked for compatibility.")).Short('a').String()
+	policyCompNodeType := policyCompCmd.Flag("node-type", msgPrinter.Sprintf("The node type. The valid values are 'device' and 'cluster'. It is required when -n is not specified. If omitted when -n is not specified, 'device' node type will be assumed.")).Short('t').String()
```

Comment:
This statement contradicts itself...Is -t required or not? I would add device as the default (with the .Default() function call as is done elsewhere in hzn cli) to this line of code and adjust the help text to match.

Reply:
Fixed